tests/affiliatedcertification/tests/affillated_certification_helm_chart.go

```diff
@@ -0,0 +1,83 @@
+	execute.BeforeAll(func() {
+		By("Define tnf config file")
+		err := globalhelper.DefineTnfConfig(
+			[]string{tsparams.TestHelmChartCertified},
+			[]string{tsparams.TestPodLabel},
+			[]string{},
+			[]string{})
+		Expect(err).ToNot(HaveOccurred(), "error defining tnf config file")
+		By("Installing helm")
+		cmd := exec.Command("/bin/bash", "-c",
+			"curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3"+
+				"&& chmod 700 get_helm.sh"+
+				"&& ./get_helm.sh")
+		err = cmd.Run()
+		Expect(err).ToNot(HaveOccurred(), "Error installing helm")
+	})
+
+	It("one helm to test,  are certified", func() {
+		By("Install a helm chart")
+		cmd := exec.Command("/bin/bash", "-c", "oc new-project affiliated-certification-helmchart-is-certified"+
+			"&& helm repo add openshift-helm-charts https://charts.openshift.io/ "+
+			"&& helm repo update && helm install example-vault1 openshift-helm-charts/hashicorp-vault")
+		err := cmd.Run()
+		Expect(err).ToNot(HaveOccurred(), "Error installing helm chart")
+
+		By("Start test")
+		err = globalhelper.LaunchTests(
+			tsparams.TestHelmChartCertified,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).ToNot(HaveOccurred(), "Error running "+
+			tsparams.TestHelmChartCertified+" test")
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TestHelmChartCertified,
+			globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+		By("remove the project")
+		cmd = exec.Command("/bin/bash", "-c", "oc delete ns affiliated-certification-helmchart-is-certified")
+		err = cmd.Run()
+		Expect(err).ToNot(HaveOccurred(), "Error delete ns affiliated-certification-helmchart-is-certified")
+	})
+
+	It("one helm to test,  are not certified", func() {
+		By("Install a helm chart")
+		cmd := exec.Command("/bin/bash", "-c", "oc new-project affiliated-certification-helmchart-is-certified"+
+			"&& helm repo add istio https://istio-release.storage.googleapis.com/charts "+
+			"&& helm repo update && oc create ns istio-system && helm install istio-base istio/base --set defaultRevision=default")
+		err := cmd.Run()
+		Expect(err).ToNot(HaveOccurred(), "Error installing helm chart")
+
+		By("Start test")
+		err = globalhelper.LaunchTests(
+			tsparams.TestHelmChartCertified,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).To(HaveOccurred(), "Error running "+
+			tsparams.TestHelmChartCertified+" test")
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TestHelmChartCertified,
+			globalparameters.TestCaseFailed)
+		Expect(err).ToNot(HaveOccurred())
+		By("remove the project")
+		cmd = exec.Command("/bin/bash", "-c", "oc delete ns affiliated-certification-helmchart-is-certified istio-system")
```

Comment:
For commands like these, shouldn't we be using the k8s library commands instead of passing in `oc` commands?  I think we should try and stick to the k8s library as much as possible.